src/common.ts

```diff
@@ -63,6 +63,8 @@ export enum ContractAddressId {
   partner = "PARTNER_TRACKER",
   zapperZapIn = "ZAPPER_ZAP_IN",
   zapperZapOut = "ZAPPER_ZAP_OUT",
```

Comment:
Are these still used?

Reply:
Checked that they are not, at least not in the SDK. There are plenty of other references to Zapper, do we want to remove those in a separate PR?

Reply:
yes, lets do a cleanup on another PR